pyHSICLasso/hsic_lasso.py

```diff
@@ -78,6 +72,14 @@ def hsic_lasso(X_in, Y_in, y_kernel, x_kernel='Gauss', n_jobs=-1, discarded=0, B
     index = np.arange(n)
     st = 0
     ed = B**2
+
+    # Normalize data
+    if x_kernel == 'Gauss':
+        X_in = X_in / (X_in.std(0) + 10e-20) * np.sqrt(float(B - 1) / B)
```

Comment:
It seems the current normalization is to normalize with respect to columns. But, we should normalize it with rows

X_in = X_in / (X_in.std(1)[:, None] + 10e-20) 

Lets' get rid of  np.sqrt(float(B - 1) / B) from this version. This is no longer important.

Please double check the standard deviation of each dimension of X_in is  1.